python-checks/src/test/java/org/sonar/python/checks/quickfix/PythonQuickFixVerifier.java

```diff
@@ -54,14 +56,17 @@ public static void verify(PythonCheck check, String codeWithIssue, String codeFi
 
     assertThat(issue.getQuickFixes())
       .as("Number of quickfixes")
-      .overridingErrorMessage("Expected 1 quickfix but found %d", issue.getQuickFixes().size())
-      .hasSize(1);
+      .overridingErrorMessage("Expected %d quickfix but found %d", codesFixed.length, issue.getQuickFixes().size())
+      .hasSize(codesFixed.length);
+
+    IntStream.range(0, codesFixed.length).forEach(index -> {
+      String codeQFApplied = PythonQuickFixVerifier.applyQuickFix(codeWithIssue, issue.getQuickFixes().get(index));
+      assertThat(codeQFApplied)
+        .as("Application of the quickfix")
+        .overridingErrorMessage("The code with the quickfix applied is not the expected result : %s instead of %s", codeQFApplied, codesFixed[index])
+        .isEqualTo(codesFixed[index]);
+    });
```

Comment:
```suggestion
    List<String> appliedQuickFix = issue.getQuickFixes().stream()
      .map(quickFix ->  applyQuickFix(codeWithIssue, quickFix))
      .collect(Collectors.toList());

    assertThat(Arrays.asList(codesFixed))
      .as("Application of the quickfix")
      .overridingErrorMessage("The code with the quickfix applied is not the expected result.")
      .isEqualTo(appliedQuickFix);
```
IMO the IntStream is quite unhandy. You can simply compare these two lists of expected and actual code.
(If you apply this suggestion you also have to change PythonQuickFixVerifierTest::one_issue_one_qf_wrong_fix because the message will change)